Behavioral/Command/Tv/Receiver.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace DesignPatterns\Behavioral\Command\Tv;
+
+class Receiver
+{
+    public $currentChannel = 0;
+
+    /**
+     * Turn On
+     */
+    public function turnOn()
+    {
+        echo "The television is on.";
```

Comment:
I think it's a bad way to show an example by generating output in class methods. This is not a clean way to program and therefore it's a not a good way to show an example here.